lib/media/track/videotrack.js

```diff
@@ -293,7 +293,11 @@ class VideoTrack extends MediaTrack {
           this._restartProcessor();
         }
       };
-      this.mediaStreamTrack.addEventListener('unmute', this._unmuteHandler);
+      if (this.mediaStreamTrack.addEventListener) {
+        this.mediaStreamTrack.addEventListener('unmute', this._unmuteHandler);
+      } else {
+        this.mediaStreamTrack.onunmute = this._unmuteHandler.bind(this);
```

Comment:
Can you also wrap the `mediaStreamTrack.removeEventListener` counterpart on line 504?

Reply:
Sure, thank you